graph.py

```diff
@@ -0,0 +1,92 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import math
+import random
+
+#Input
+
+print("Please choose type of function\n"
```

Comment:
Maybe you should think about more user-friendly interface than simple CLI. Try pyQt.